Source/Renderer/createUniform.js

```diff
@@ -167,7 +169,9 @@ define([
                 this._gl.uniform4f(this._location, v.x, v.y, v.z, v.w);
             }
         } else {
+            //>>includeStart('debug', pragmas.debug);
```

Comment:
This is how we normally do it, I don't think the empty else block incurs any sort of performance penalty and this way makes the code cleaner.

The other option without having to repeat anything would be to write:
```javascript
}
//>>includeStart('debug', pragmas.debug);
else {
}
````

Empty else blocks are not removed at minification time, but I imagine might get removed by the optimizer during browser compilation, but I wouldn't worry about it unless something showed up in the profile.